src/app/start-guide/what-is-markdown/page.md

```diff
@@ -1,112 +1,105 @@
-Please visit [my homepage](https://www.craftz.dog/)
+## Why should I try it? 🔮
 
-- List item 1
-- List item 2
-```
+Imagine you're an architect. You have ideas brimming in your mind, designs waiting to take shape.
+Now, you could sketch these on any piece of paper, but you choose a drafting table.
+Why? Because it's tailored for your work.
+It has the right tools and features for productive work.
+With Markdown, you don't write notes, you craft them. And, Inkdrop is the best tool for crafting.
 
-{% /snippet %}
+## What can I do with Markdown? 📝
 
-But if you're a programmer or diving into the tech world, you might often hear about "[GitHub-flavored Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)" or GFM for short.
-GFM is created by [GitHub](https://github.com/), the world's largest online software development platform, for its users to help them write better documentation and discuss technical topics more effectively.
-It supports the following additional features:
+Inkdrop lets you format text in many ways. Here are some basic examples:
 
-- **Code Blocks**: With GFM, you can easily insert blocks of code by wrapping them with three backticks (\```). This not only differentiates the code from the rest of the text but also applies syntax highlighting based on the language of the code.
-- **Task Lists**: Easily track to-dos using checkboxes.
+* `# Headings`. The more `#` you use, the smaller the heading. `## Heading` gives you a second-level heading.
+* \**Italicized*\* and \*\*bold\*\* text-> *Italicized* and **bold** text.
+* \[Link](https://www.craftz.dog/) -> [Link](https://www.craftz.dog/).
+* \~~Line-through~\~ -> ~~Line-hrough~~.
+* \`Inline code\` -> `Inline code`.
+
+That's not all, though.
 
-For example:
+### Advanced formatting with GitHub-flavored Markdown 🤓 
 
-{% snippet lang="md" filename="github-flavored.md" %}
+If you're a programmer or diving into the tech world, you might have heard about [GitHub-flavored Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) (GFM for short). This is a superset of Markdown. It includes all the features of Markdown and adds a few more to meet day-to-day needs of developers.:
 
+- **Fenced code Blocks**: With GFM, you can easily insert code blocks by wrapping them with three backticks (\```). You can also specify a specific programming language for correct syntax highlighting.
+{% snippet lang="md" filename="Code blocks" %}
 ````
 \```js
 function helloWorld() {
   console.log("Hello, world!");
 }
 \```
-
-\```python
-def hello_world():
-    print("Hello, world!")
-
-hello_world()
-\```
-
+````
+{% /snippet %}
+- **Task Lists**: Easily track to-dos using checkboxes.
+{% snippet lang="md" filename="To-dos" %}
+````
 - [ ] Task item 1
 - [x] Completed task item 2
 ````
-
+{% /snippet %}
```

Comment:
I'm planning to cover Mermaid and Math on another page because it requires installing plugins.
Also, it is a bit overwhelming to mention the advanced syntaxes for beginners at this point.
Just simply explain why Markdown is good for tech note-taking on this page.